### ISSUE TITLE
[Fleet]: `Package` term is used instead of `integration` under Windows integration overview page.

### ISSUE THREAD

**Issue by amolnater-qasource** [elastic/integrations#2468]
**Kibana version:**  8.1 Snapshot Kibana cloud environment

**Host OS and Browser version:** All , All

**Build details:**
```
BUILD: 49380
COMMIT: 87f753dba0515c23a1cc4bdf6feac2df2200baea
```

**Preconditions:**
1. 8.1 Snapshot Kibana cloud environment should be available.

**Steps to reproduce:**
1. Login to Kibana environment.
2. Navigate to Integrations>Windows integration.
3. Observe packages term is used instead of integration under overview page.

**Screenshot:**
![8](https://user-images.githubusercontent.com/77374876/148013675-ba64cb9d-cfc1-4fe2-b430-c63b396fc968.JPG)

**Expected Result:**
`Integration` term should be used instead of `package` under Windows integration overview page.

**What's working fine:**
- for System integration:
![9](https://user-images.githubusercontent.com/77374876/148013801-2ff513d9-41a3-4dc9-915f-c059cfc794ab.JPG)

- for Endpoint Security integration:
![10](https://user-images.githubusercontent.com/77374876/148013901-ebf01b47-a8da-47c4-9571-81748ef550c8.JPG)

**Comment by akshay-saraswat** [comment]
I agree. Windows integration must be presented as an integration instead of a package.

**Comment by jillguyonnet** [comment]
@kpollich @jen-huang It seems this was fixed in https://github.com/elastic/integrations/pull/3352. The current version of the Windows integration (1.16) has the correct terminology. Am I OK to close this issue?
